src/sv-pipeline/scripts/make_evidence_qc_table.py

```diff
@@ -0,0 +1,274 @@
+    df_non_diploid = read_non_diploid(filename_cnv_qvalues)
+    df_manta_high_outlier = read_outlier(filename_high_manta, get_col_name("manta", "high"))
+    df_melt_high_outlier = read_outlier(filename_high_melt, get_col_name("melt", "high"))
+    df_wham_high_outlier = read_outlier(filename_high_wham, get_col_name("wham", "high"))
+    df_total_high_outliers = read_all_outlier(filename_high_manta, filename_high_melt, filename_high_wham, "high")
+    df_manta_low_outlier = read_outlier(filename_low_manta, get_col_name("manta", "low"))
+    df_melt_low_outlier = read_outlier(filename_low_melt, get_col_name("melt", "low"))
+    df_wham_low_outlier = read_outlier(filename_low_wham, get_col_name("wham", "low"))
+    df_total_low_outliers = read_all_outlier(filename_low_manta, filename_low_melt, filename_low_wham, "low")
+
+    # all data frames
+    dfs = [df_ploidy, df_bincov_median, df_wgd_scores, df_non_diploid, df_manta_high_outlier,
+           df_melt_high_outlier, df_wham_high_outlier, df_total_high_outliers,
+           df_manta_low_outlier, df_melt_low_outlier, df_wham_low_outlier, df_total_low_outliers]
+    output_df = reduce(
+        lambda left, right: pd.merge(left, right, on=ID_COL, how="outer"), dfs
+    ).fillna(np.nan)  # "none"
+    output_df = output_df[output_df[ID_COL] != EMPTY_OUTLIERS]
+
+    # save the file
+    output_df.to_csv(f"{output_prefix}.evidence_qc_table.tsv", sep="\t", header=True, index=False)
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-y", "--estimated-copy-number-filename",
+        help="Sets the filename containing estimated copy numbers per contig.", nargs=1)
+
+    parser.add_argument(
+        "-d", "--median-cov-filename",
+        help="Sets the filename containing median coverage.", nargs=1)
+
+    parser.add_argument(
+        "-g", "--wgd-scores-filename",
+        help="Sets the filename containing WGD scores.", nargs=1)
+
+    parser.add_argument(
+        "-b", "binwise-cnv-qValues-filename",
+        help="Sets the filename containing bin-wise CNV q-values.", nargs=1)
+
+    parser.add_argument(
+        "-z", "--manta-qc-outlier-high-filename",
+        help="Sets the filename containing Manta QC outlier high.", nargs=1)
+
+    parser.add_argument(
+        "-e", "--melt-qc-outlier-high-filename",
+        help="Sets the filename containing Melt QC outlier high.", nargs=1)
+
+    parser.add_argument(
+        "-w", "--wham-qc-outlier-high-filename",
+        help="Sets the filename containing Wham QC outlier high.", nargs=1)
+
+    parser.add_argument(
+        "-a", "--manta-qc-outlier-low-filename",
+        help="Sets the filename containing Manta QC outlier low.", nargs=1)
+
+    parser.add_argument(
+        "-s", "--melt-qc-outlier-low-filename",
+        help="Sets the filename containing Melt QC outlier low.", nargs=1)
+
+    parser.add_argument(
+        "-w", "--wham-qc-outlier-low-filename",
```

Comment:
-w is used twice as a short argument name, here and for wham high outlier
```suggestion
        "-q", "--wham-qc-outlier-low-filename",
```

Reply:
very nice catch! I fixed it [here](https://github.com/broadinstitute/gatk-sv/blob/sz_evidence_qc_python_script/src/sv-pipeline/scripts/make_evidence_qc_table.py#:~:text=%22%2Dr%22%2C%20%22%2D%2Dwham%2Dqc%2Doutlier%2Dlow%2Dfilename%22%2C).